packages/cli/test/integration/generators/copyright.integration.js

```diff
@@ -68,15 +68,15 @@ describe('lb4 copyright', /** @this {Mocha.Suite} */ function () {
 
     assertHeader(
       ['src/application.ts', 'lib/no-header.js'],
-      `// Copyright ACME Inc. ${year}. All Rights Reserved.`,
+      `// Copyright ACME Inc. 2020,${year}. All Rights Reserved.`,
```

Comment:
This is correct. I believe we are using the following algorithm:

- If the file was created in the current year (e.g. `2021`), then we put only that year to the copyright notice (`2021`).
- If the file was created earlier, then we put the year when it was created and the current year (e.g. `2020,2021`).

This means that after changing existing files in this year, we will eventually have to update the copyright notices from `2020` to `2020,2021`, `2019,2020` to `2019,2021` and so one.